include/dlpack/dlpack.h

```diff
@@ -54,10 +45,19 @@ typedef struct {
   /*! \brief The device index */
   int device_id;
   /*! \brief The device type used in the device. */
-  int device_type;
+  DLDeviceType device_type;
 } DLContext;
 
 /*!
+ * \brief The type code options DLDataType.
+ */
+typedef enum {
+  kInt = 0,
+  kUInt = 1,
+  kFloat = 2,
+} DLDataTypeCode;
```

Comment:
keep the 'u' suffix so it is unsigned, as the code is unsigned, to prevent compiler warning